autograder/submitty_autograding_shipper.py

```diff
@@ -796,18 +808,23 @@ def checkout_vcs_repo(config, my_file, which_branch):
         checkout_log_file = os.path.join(results_path, "logs", "vcs_checkout.txt")
 
         # -----------------------------------------------------------
-        # NOTE: THIS MAY EVENTUALLY BE DEPRECATED AND REMOVED
-        # If the specified branch does not exist (or the repo does not exist)
+        # First see if the course-specified branch exists
         ls_branch_command = [
             '/usr/bin/git', 'ls-remote', '--exit-code', vcs_path, which_branch
         ]
         try:
             subprocess.check_call(ls_branch_command)
         except subprocess.CalledProcessError:
-            # except subprocess.CalledProcessError as error:
-            # then try the deprecated master branch instead
-            which_branch = 'master'
-        # END DEPRECATED NOTE
+            # Then try to use the 'main' branch -- the recommended default
+            # starting in 2020
```

Comment:
I take it we are going to no longer deprecate master as the default at some point?

Reply:
Yeah, I think we leave this here indefinitely.  If students use non-submitty hosted repos they may still be defaulted to master.